File: aisile-manager-web/src/main/webapp/js/service/brandService.js
		 //品牌服务层
		 app.service('brandService',function($http){
			    	//读取列表数据绑定到表单中
			    	this.findAll=function(){
			    		return $http.get('../brand/findAll.do');		
			    	}
			    	//分页
			    	this.findPage=function(page,rows){
			    		return $http.get('../brand/findPage.do?page='+page+'&rows='+rows);		
			    	}
			    	//保存
			    	this.save = function(methodName,entity){
			    		return $http.post('../brand/'+ methodName +'.do',entity);
			    	}
			    	//查询
			    	this.findOne = function(id){
			    		return $http.get('../brand/findOne.do?id='+id);
			    	}
			    	//删除
			    	this.dele = function(selectIds){
			    		return $http.get('../brand/delete.do?ids='+selectIds);
			    	}
			    	//搜索
			    	this.search=function(page,rows,searchEntity){
			    		return $http.post('../brand/search.do?page='+page+"&rows="+rows,searchEntity);
			    	}
			    	//下拉列表数据
			    	this.selectOptionList=function(){
			    		return $http.get('../brand/selectOptionList.do');
			    	}
		 }); 

File: aisile-sellergoods-interface/src/main/java/com/aisile/sellergoods/service/BrandService.java
package com.aisile.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.aisile.pojo.TbBrand;
import com.aisile.pojo.entity.PageResult;

public interface BrandService {
	public List<TbBrand> findAll();
	public PageResult findPage(int pageNum,int pageSize);
	public void add(TbBrand brand);
	public void update(TbBrand brand);
	public TbBrand findOne(Long id);
	public void delete(Long [] ids);
	public PageResult findPage(TbBrand brand, int pageNum,int pageSize);
	public List<Map> selectOptionList();
}
